src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripQuery.java

```diff
@@ -432,6 +445,23 @@ public static GraphQLFieldDefinition create(
           .defaultValue(preferences.itineraryFilter().debug())
           .build()
       )
+      .argument(
+        GraphQLArgument
+          .newArgument()
+          .name("relaxTransitSearchCostCriteria")
+          .description(
+            """
+            Whether non-optimal transit paths should be returned.
```

Comment:
This is outside the scope of this PR but I think we should introduce a mechanism so that these long form explanations can be shared between different APIs and the config mapper. WDYT?

Reply:
That is a good idea. But we need to think how to do this. We can plug in a Resource in the config generation, and look up the `description` (if not explisit set), but in the GraphQL API we want to use schemas. We can probably do resource filtering on the schema, using the same properties file as we used for the description.

Reply:
Yes, definitely